### ISSUE TITLE
Google assistant Jarvis

### ISSUE THREAD

**Issue by landaisbenj** [alexylem/jarvis#863]
Hello.

Je suppose qu'avec le SDK de google assistant, publié par Google, il serait possible de rendre Jarvis plus attrayant. Je sais qu'un des plus belle réussite de jarvis est d’être hors ligne, mais un de ces défauts est de ne pas etre "casual".

Le google home mini est attrayant, simple et matériellement remarquable pour son prix. Et bien qu'il comporte beaucoup d'option sympas lui aussi, il n'est pas Jarvis. Et il n'a pas de TTS. J'utilisai Jarvis surtout pour le TTS et pour domoticz. Mais avec GA, il faut dire que c'est un peu passé à la trappe pour moi (et je suppose pour beaucoup d'entre nous). Si seulement je pouvais combiner les deux (les capacités technique du GH et sa grande connaissance avec les fonctionnalités de Jarvis et son API avec TTS) alors je serais surement le plus gatté.

Je pense qu'il est largement possible de créé un lien entre Jarvis et le sdk de google assistant. Peut être avec une phrase clef ("Jarvis, dis a google, quelles sont les actualités") ou en total autonomie, si jarvis ne trouve pas d'action alors il voit du coté du SDK...

Il ne suffit que d’intégrer le sdk GA (python) en plugin. Et de créer la passerelle entre les deux.

Est-ce que cette idée en est une bonne ? Ou est-ce que cela vous rebute ? 
Je sais que les plugins sont libres, mais je n'ai pas forcement les capacités de bien le faire. 
Et c'est pour cela que j'en parle avant.

Si un dev veux se pencher dessus tant mieux. Avec des connaissances en python et Jarvis ce sera très simple (j'ai réussi moi même à monter mon propre GA sur raspberry, sans connaissance Python). 
Si personne ne se penche dessus, mais que vos remontés sont bonnes alors je pourrait regardé.

En tout cas, c'est une idée dans l'optique de faire respirer Jarvis qui fonctionne beaucoup trop bien tout seul et peut être de lui donner plus de visibilité.

**Comment by Oliv4945** [comment]
La dernière fois que j'ai essayé il n'y avait que très peu de fonctions intégrées, ça a peut être changé. 
Si vous voulez faire ça il vaut mieux partir sur l'Alexa ;)

**Comment by landaisbenj** [comment]
Oui, je n'avais pas non plus trouvé l'équivalence entre le sdk et l'assistant en lui même. Je proposais ga parce que je l'ai et que j'ai aussi déjà fait le montage. Mais bien sûr si cela est possible autant faire les deux!!!
